data/json/items/gun/shot.json

```diff
@@ -899,6 +889,131 @@
     "//": "no provision for separate triggers",
     "pocket_data": [ { "pocket_type": "MAGAZINE", "rigid": true, "ammo_restriction": { "shot": 2 } } ]
   },
+  {
+    "id": "ar12_shotgun",
+    "copy-from": "shotgun_pump",
+    "looks_like": "remington_870",
+    "type": "GUN",
+    "name": { "str": "rifle shotgun" },
+    "description": "Despite looking like a military-issued rifle, it is, in fact, a shotgun.  Accept variery of magazines.",
```

Comment:
```suggestion
    "description": "Despite looking like a military-issued rifle, this is, in fact, a shotgun.  It can accept a variety of magazines.",
```